src/allotropy/parsers/chemometec_nc_view/chemometec_nc_view_structure.py

```diff
@@ -0,0 +1,54 @@
+from allotropy.allotrope.models.shared.definitions.definitions import JsonFloat
+from allotropy.allotrope.schema_mappers.adm.cell_counting.benchling._2023._11.cell_counting import (
+    Measurement,
+    MeasurementGroup,
+    Metadata,
+)
+from allotropy.parsers.chemometec_nc_view import constants
+from allotropy.parsers.constants import NOT_APPLICABLE
+from allotropy.parsers.utils.pandas import SeriesData
+from allotropy.parsers.utils.uuids import random_uuid_str
+from allotropy.parsers.utils.values import try_float_or_nan
+
+
+def create_metadata(data: SeriesData, file_name: str) -> Metadata:
+    return Metadata(
+        file_name=file_name,
+        software_name=constants.SOFTWARE_NAME,
+        device_type=constants.DEVICE_TYPE,
+        equipment_serial_number=data[str, "INSTRUMENT"].split(":")[-1].strip(),
+        product_manufacturer=constants.PRODUCT_MANUFACTURER,
+        detection_type=constants.DETECTION_TYPE,
+        model_number=NOT_APPLICABLE,
+        unc_path=NOT_APPLICABLE,
+    )
+
+
+def create_measurement_groups(data: SeriesData) -> MeasurementGroup:
+    return MeasurementGroup(
+        analyst=data[str, "OPERATOR"],
+        measurements=[
+            Measurement(
+                measurement_identifier=random_uuid_str(),
+                processed_data_identifier=random_uuid_str(),
+                timestamp=_format_timestamp(data[str, "NAME"]),
+                sample_identifier=data[str, "SAMPLE ID"],
+                viability=data[float, "VIABILITY (%)"],
+                total_cell_density=_format_unit(data[str, "TOTAL (cells/ml)"]),
```

Comment:
Total cell density is not required in the ASM document, for that reason, we should make this `data.get`

I believe this probably applies for some other fields in this block.

Generally, if the `Measurement` field is not required, the retrieval should not be required.